src/engine/prague.md

```diff
@@ -122,10 +63,11 @@ The response of this method is updated with [`ExecutionPayloadV4`](#ExecutionPay
 #### Response
 
 * result: `object`
-  - `executionPayload`: [`ExecutionPayloadV4`](#ExecutionPayloadV4)
+  - `executionPayload`: [`ExecutionPayloadV3`](./cancun.md#executionpayloadv3)
   - `blockValue` : `QUANTITY`, 256 Bits - The expected value to be received by the `feeRecipient` in wei
   - `blobsBundle`: [`BlobsBundleV1`](#BlobsBundleV1) - Bundle with data corresponding to blob transactions included into `executionPayload`
   - `shouldOverrideBuilder` : `BOOLEAN` - Suggestion from the execution layer to use this `executionPayload` instead of an externally provided one
+  - `executionRequestsHash`: `DATA`, Bytes 32 - Hash of execution layer triggered requests obtained from the `executionPayload` transaction execution.
```

Comment:
Won't we need the actual execution requests here to put into the beacon block body?!